wolox-books/src/styles.scss

```diff
@@ -0,0 +1,8 @@
+@import './scss/equalizer.scss';
+@import './scss/form.scss';
+@import './scss/colors';
```

Comment:
delete color here, because colors is a partial and hasn't any sense here

https://sass-lang.com/guide
```
Partials
You can create partial Sass files that contain little snippets of CSS that you can include in other Sass files. This is a great way to modularize your CSS and help keep things easier to maintain. A partial is a Sass file named with a leading underscore. You might name it something like _partial.scss. The underscore lets Sass know that the file is only a partial file and that it should not be generated into a CSS file. Sass partials are used with the @use rule.
```